mocha-multi.js

```diff
@@ -293,6 +293,14 @@ function mochaMulti(runner, options) {
 
 class MochaMulti {
   constructor(runner, options) {
+    debugger;
+    if (options.mochaMultiOptions) {
+      let reporterOptions = options.mochaMultiOptions;
+      if (typeof reporterOptions === 'string') {
+        reporterOptions = JSON.parse(reporterOptions);
+      }
+      Object.assign(options, { reporterOptions });
```

Comment:
Maybe output a warning of `options.reporterOptions` is already a non-empty object, as it'll be overwritten?

Reply:
Hmmmmm - my gut says if you've gone out of your way to use the "mochaMultiOptions" key then overwriting is the behaviour you're opting into so I din't think a warning makes much sense?

What I was just thinking though it a plain assignment might be better that `Object.assign()` though.

Reply:
You're right that it's a perfectly reasonable outcome that `mochaMultiOptions` takes precedence, it just seems like a minor footgun.

Reply:
Think this is better as of: https://github.com/peakon/mocha-multi/pull/1/files#diff-a12c9c4ac2eeaf73fc7d38740c2da43a66cad7a74bcf4b95ef44955868f5a237R302